### DISCUSSION TITLE
presets externals should have "allowlist"

### DISCUSSION THREAD

**Discussion by Saodd** [webpack/webpack#17892] [Q&A]
## Feature request

presets externals should have "allowlist", to force bundle some libs that excluded by target defaults.

**What is the expected behavior?**

for example, 
{
  externals: { 'events': false }
}
to force 'events' bundled in rather than excluded.

**What is motivation or use case for adding/changing the behavior?**

Case:
I'm using webpack@5.88.2.
I have a webpack-config. Its target is "electron19-renderer". 
When I try to use "events" lib, I find the "events" is excluded from being bundled by default. In ouput js-file it is compiled as "require('events')". But actually I expect my code to run in  electron-renderer -- it is browser environment and doesn't have "require" function or 'events' in global.
I tried a lot, but I couldn't find a good solution. 

**How should this be implemented in your opinion?**

**Are you willing to work on this yourself?**
sorry, the "ExternalsPlugin" is too complex for me to work on myself.

**Comment by alexander-akait** [comment]
Sorry for delay, can you provide the configuration, I want to see your `target`, `events: false` means ignore module
